Jenkinsfile

```diff
@@ -11,6 +11,7 @@ pipeline {
   }
   triggers {
     pollSCM('H/5 * * * *')
+    cron('@daily')
```

Comment:
Commit message states hourly. Also, I suspect this will be at the same time every day for every job. We might want to use 'H H * * *' instead to spread jobs throughout the day.

Reply:
Fixed in https://github.com/mozilla/stubattribution-tests/pull/126.

Also, first results show it's working as expected:

https://qa-master.fxtest.jenkins.stage.mozaws.net/job/stubattribution.prod/2353/ and
https://qa-master.fxtest.jenkins.stage.mozaws.net/job/stubattribution.prod/2353/pollingLog/